scalableminds/webknossos
# Vx followup

- Optimizes inserts, when many events are sent in a single request, by grouping them into single database queries
- Optimizes workflow listing by batching database queries
- Provides consolidated view of tasks that have been resumed, i.e. combining the information about the chunks of multiple runs
- Removes some cyclic dependencies in the frontend
- Adds support for Enumerations in SqlInterpolation (stolen from #6723)

### Steps to test:
- Run a voxelytics workflow with multiple chunks, interrupt the run and resume
- Once complete, the workflow report should provide accurate information about all completed chunks

------
(Please delete unneeded items, merge only when none are left open)
